### ISSUE TITLE
Plotmanager running, but not adding Plots

### ISSUE THREAD

**Issue by WasHupDa** [swar/Swar-Chia-Plot-Manager#501]
Hey all,

I'm sure I set something wrong in the config.yaml, but I hope someone can tell me what :).
I'm using Ubuntu Server 20.04

The config file looks like this:

chia_location: /home/USERNAME/chia-blockchain/venv/bin/chia
folder_path: /home/USERNAME/.chia/mainnet/log

- name: SSD1
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: /media/ssd_1/Plots
    temporary2_directory:
    destination_directory: /media/trustno1/Farm_1/Farms
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 4
    max_concurrent_with_start_early: 4
    stagger_minutes: 60
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

The Manager is starting properly and the View shows that it is running, it's just not adding any plot.
Did I forget to configure something?

Appreciate your help
Peter

**Comment by WasHupDa** [comment]
Nevermind, I think I found the error:
Instead of 
folder_path: /home/USERNAME/.chia/mainnet/log

it should be
folder_path: /home/USERNAME/.chia/mainnet/plotter.

**Comment by WasHupDa** [comment]
I saw that instead of 
folder_path: /home/USERNAME/.chia/mainnet/log

it should be
folder_path: /home/USERNAME/.chia/mainnet/plotter.
But still not adding plots automatically.

**Comment by razz4t7** [comment]
im having the same problem
in my log files it says
"Second instance. Quitting."
same shit after reboot

**Comment by gitrid** [comment]
I'm having exactly the same issue as @WasHupDa, RuntimeError: Please initialize (or migrate) your config directory with 'chia init'.

What I investigated more: I believe that it can be an issue with permissions on ssh connection. When I do chia init (placed in /home/chia-blockchain) it confirms that it was already done: 
```
Chia directory /home/bro/.chia/mainnet
No keys are present in the keychain. Generate them with 'chia keys generate'
/home/bro/.chia/mainnet already exists, no migration action taken
```
However "/home/bro/.chia/mainnet" looks different on my Ubuntu Server than on my Ubuntu PC.

Server (where the problem occurs):
![image](https://user-images.githubusercontent.com/21285019/118600399-3b7bde00-b7b1-11eb-8a1f-18959b0df61b.png)
Ubuntu PC (everything works):
![image](https://user-images.githubusercontent.com/21285019/118600496-5cdcca00-b7b1-11eb-9f4f-8799c85864f6.png)

I've tried different chmod 777 but nothing has helped. 

@swar have you faced this before?

**Comment by WasHupDa** [comment]
Hey there,

when I'm running the chia init, I get the message:

WARNING: using a farmer address which we don't have the private keys for.
WARNING: using a pool address which we don't have the private keys for.
/home/USERNAME/.chia/mainnet already exists, no migration action taken.

Directory looks like this:
![Plotman_Issue](https://user-images.githubusercontent.com/28943670/118627810-a6b5b800-b791-11eb-8254-7e26bed290c8.png)

**Comment by gitrid** [comment]
But does it work for you? These warnings are ok if you are passing keys via swar config, my issue seems to be different

**Comment by RyanWarwick** [comment]
Hi, 
From what I observe and understand, running ```chia init``` creates the .chia directory in the current active user's (the one who ran the command) $HOME directory, which is usually /home/$USERNAME. I have no issues when running ```chia init``` followed by 
```. ./venv/bin/activate```
```python3 manager.py start```
```python3 manager.py view``` accordingly. 

Are you running the ```manager.py``` as the same user who ran ```chia init```?

**Comment by WasHupDa** [comment]
> But does it work for you? These warnings are ok if you are passing keys via swar config, my issue seems to be different

No, unfortunately it doesn't add any plot to the list, because of the runtime error, which is generated in the chia plots.py.
At the moment I have no idea, why the root path should not exist.

Greetings,
Peter

**Comment by RyanWarwick** [comment]
> > Are you running the manager.py as the same user who ran chia init?
> 
> Thanks a lot @RyanWarwick, this was solving the issue!
> 
> It looks like the Swar-Chia-Plot-Manager-main directory was owned by root user.
> So I did a
> 
> **sudo chown -hR USERNAME ./Swar-Chia-Plot-Manager-main/**
> 
> and now everything is working perfectly!!
> 
> Greetings,
> Peter

Nice, that's great to hear :)

**Comment by swar** [comment]
I hope people look for this issue via a search because you've found the answer! Good work.

**Comment by racabu** [comment]
I'm doing this and the issue still the same:

  File "/usr/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
PermissionError: [Errno 13] Permission denied: '/home/usr/chia-blockchain/chia/cmds'
